pymoliere/construct/embedding_util.py

```diff
@@ -93,6 +93,7 @@ def embed_records(
     text_field:str,
     max_sequence_length:int,
     out_embedding_field:str="embedding",
+    show_pbar:bool=False,
```

Comment:
Note: Its important to always have optional boolean arguments default to "false."
This is because conceptually, it makes more sense to turn things "on" than to disable things.